server/lib/referenceLookup.js

```diff
@@ -59,3 +78,31 @@ function indexIdentifiers() {
   });
 }
 
+// hack: handle special case of stock movements
+function stockMovementType(id) {
```

Comment:
A better name would be `getStockMovementType()` since this is going to return something.